### PR TITLE
Prepare v0.44.0 development

### PR DESCRIPTION
**Pull request by philipaconrad**
:tada:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -106,6 +106,8 @@ This is a bug fix release that addresses the following:
 2. Policies owned by a bundle could be replaced via the REST API because of a missing bundle scope check ([#4846](https://github.com/open-policy-agent/opa/issues/4846)).
 3. Adds missing `future.keywords` import for the examples in the policy testing section of the docs ([#4849](https://github.com/open-policy-agent/opa/issues/4849)), reported by @robert-elles.
 
+## Unreleased
+
 ## 0.42.0
```
Comment (ashutosh-narkar) [inline CHANGELOG.md:111]: This should be `0.43.0`

Comment (philipaconrad) [reply CHANGELOG.md:111]: What happened there? I'll push up a fix momentarily.

Comment (philipaconrad) [reply CHANGELOG.md]: Good catch! I'm not sure how, but either I mis-applied the patch, or *something* went amiss.

I've manually corrected it.

**Review by ashutosh-narkar — APPROVED** [review]

**Comment by philipaconrad** [comment]
Awesome! :smile: Thanks for the review, @ashutosh-narkar!

Will merge when green.

**Pull request merged**